toss/fitness/fitness_function_utils.py

```diff
@@ -64,49 +65,69 @@ def compute_space_coverage(positions, velocities, timesteps, radius_outer_boundi
         positions (np.ndarray): (3,N) Array of positions (in cartesian coordinates).
         velocities (np.ndarray): (3,N) Array of positions (in cartesian frame).
         timesteps (np.ndarray): (N) Array of time values for each position.
-        radius_outer_bounding_sphere (float): Radius of outer bounding sphere.
+        radius_min (float): Inner radius of spherical grid, typically radius_inner_bounding_sphere.
+        radius_max (float): Outer radius of spherical grid, typically radius_outer_bounding_sphere.
 
     Returns:
         ratio (float): Number of True values to the total number of values in the boolean array
     """
 
+    # Fixed maximal velocity from previously defined trajectory. 
+    # We use a fixed value to avoid prioritizing higher velocities. 
+    #
+    # NOTE: The fitness value for covered volume will not be feasible if
+    #        its maximal velocity exceeds the provided value below as the grid
+    #        spacing will be too small. Please use the scaling factor to adapt for this.
+    scaling_factor = 1
```

Comment:
I think it will be easiest to add it in a cfg file after its has been merged. The cfg file already exist in the main branch and it would be nice to have it there. Whenever we have merged this branch, I will close it only after I've added the scaling factor to the existing cfg. 👍

Reply:
just merge main into this?